{
    "config": {
        "abort": {
            "already_configured": "El dispositivo ya est\u00e1 configurado"
        },
        "error": {
            "connection_exception": "No se pudo conectar",
            "invalid_auth": "Autenticaci\u00f3n no v\u00e1lida",
            "unauthenticated_exception": "Autenticaci\u00f3n no v\u00e1lida",
            "unknown": "Error inesperado"
        },
        "step": {
            "user": {
                "data": {
                    "password": "Contrase\u00f1a",
                    "username": "Usuario"
                }
            }
        }
    }
}